.github/workflows/podman-test.yml

```
@@ -0,0 +1,207 @@
+
+          # Test container can start without audio device in CI
+          echo "Testing container startup without audio device..."
+          if podman run --rm -d \
+            --name test-no-audio \
+            -e BIRDNET_UID=1000 \
+            -e BIRDNET_GID=1000 \
+            -e BIRDNET_REALTIME_AUDIO_ENABLED=false \
+            -v ./config:/config \
+            -v ./data:/data \
+            ghcr.io/tphakala/birdnet-go:nightly; then
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)